roseau/load_flow/models/lines/parameters.py

```diff
@@ -218,10 +207,14 @@ def _sym_to_zy(
                 Phase to neutral susceptance (siemens/km)
 
         Returns:
-            The impedance and admittance matrices and the line model. The line model may be downgraded from
-            SYM_NEUTRAL to SYM if the model of the neutral is not possible.
+            The impedance and admittance matrices.
         """
-
+        # Compute some values
+        xpn_isna = pd.isna(xpn)
+        bn_isna = pd.isna(bn)
+        bpn_isna = pd.isna(bpn)
+        zn_isna = pd.isna(zn)
+        any_neutral_na = any((xpn_isna, bn_isna, bpn_isna, zn_isna))
```

Comment:
optional nit: this can be simplified

```suggestion
        # Check if all neutral parameters are valid
        any_neutral_na = any(pd.isna([xpn, bn, bpn, zn]))
```